data.sql

```diff
@@ -22,3 +22,13 @@ VALUES  (5,'Charmander', '2020-02-08',-11,FALSE,0),
         (10,'Blossom', '1998-10-13',17,TRUE,3),
         (11,'Ditto', '2022-05-14', 22,TRUE,4);
 
+/* Insert data into the owners table */
+
+INSERT INTO owners (full_name, age)                                                    
+VALUES  ('Sam Smith', 34),
+        ('Jennifer Orwell', 19),
+        ('Bob', 45),
+        ('Melody Pond', 77 ),
+        ('Dean Winchester', 14),
+        ('Jodie Whittaker', 38);
```

Comment:
* Kindly add these missing data as stated in the [project requirement](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_multiple_tables.md#:~:text=Insert%20the%20following%20data%20into%20the%20species,Dean%20Winchester%20owns%20Angemon%20and%20Boarmon.) 

> Insert the following data into the species table:
Pokemon
Digimon

> Modify your inserted animals so it includes the species_id value:
> If the name ends in "mon" it will be Digimon
> All other animals are Pokemon
> 

> Modify your inserted animals to include owner information (owner_id):
> Sam Smith owns Agumon.
> Jennifer Orwell owns Gabumon and Pikachu.
> Bob owns Devimon and Plantmon.
> Melody Pond owns Charmander, Squirtle, and Blossom.
> Dean Winchester owns Angemon and Boarmon.

Reply:
I forgot to add them to the data.sql thankyou.